decidim-proposals/app/cells/decidim/proposals/proposals_picker_cell.rb

```diff
@@ -9,45 +9,76 @@ class ProposalsPickerCell < Decidim::ViewModel
       MAX_PROPOSALS = 1000
 
       def show
-        if filtered?
-          render :proposals
-        else
-          render
-        end
+        render
       end
 
       alias component model
 
+      def form
+        options[:form]
+      end
+
+      def field
+        options[:field]
+      end
+
+      def form_name
+        "#{form.object_name}[#{method_name}]"
+      end
+
+      def method_name
+        field.to_s.sub(/s$/, "_ids")
+      end
+
+      def selected_ids
+        form.object.send(method_name)
+      end
+
+      def proposals
+        @proposals ||= Decidim.find_resource_manifest(:proposals).try(:resource_scope, component)
+                         &.includes(:component)
+                         &.published
+                         &.not_hidden
+                         &.order(id: :asc)
+      end
+
+      def decorated_proposals
+        proposals.limit(MAX_PROPOSALS).each do |proposal|
+          yield Decidim::Proposals::ProposalPresenter.new(proposal)
+        end
+      end
+
+      # deprecated
```

Comment:
I would prefer to remove, either to add a `ActiveSupport::Deprecation.warn("...")` message. 

Same comment valid for all the below `#deprecated` messages

Reply:
Nice! I didn't know it about such clause (in JS-doc in `@deprecated`). I'll apply there.

Reply:
I rather go with full removal of all the related files (there's a bunch). Let's see then if the specs agree with me :)